golem-worker-executor-base/src/grpc.rs

```diff
@@ -141,6 +142,7 @@ pub struct WorkerExecutorImpl<
 > {
     /// Reference to all the initialized services
     services: Svcs,
+    worker_fork: Arc<dyn WorkerFork + Sync + Send>,
```

Comment:
The idea of `HasAll` is that it has all the services. So you should add the new one to those traits according to how it is done for the existing services, and then you don't need a new field here.

Reply:
@vigoo  I tried this (a few days ago), but if I remember correctly, the worker_fork implementation pretty much needs of all what's in HasAll.  So I was a bit confused in this area. I think worker_fork has the same type of behavior as `DefaultWorkerActivator` in terms of dependency requirements.

Reply:
I think it must be similar to `Rpc` (it does both local and remote worker invocations).

This change is a must to keep consistency of how worker executor is built.